Annotato/Annotato/ViewController.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  ViewController.swift
+//  Annotato
+//
+//  Created by Sivayogasubramanian on 10/3/22.
+//
+
+import UIKit
+import PDFKit
+import UniformTypeIdentifiers
+
+class ViewController: UIViewController {
+    let pdfView = PDFView()
+    var pdfTapGestures: [UITapGestureRecognizer] = []
+    var tripleTapAnnotateGesture = UITapGestureRecognizer()
+    @IBOutlet private var nextButton: UIBarButtonItem!
+    @IBOutlet private var importFilesButton: UIBarButtonItem!
+
+    let buttonHeight = 50.0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addPdfView()
+        addObservers()
+        addTripleTapGestureAddAnnotation()
+        initializeGoToDocumentViewButton()
+    }
+
+    private func initializeGoToDocumentViewButton() {
+        let documentViewButton = FilledButton(
+            title: "Doc List",
+            frame: CGRect(x: .zero, y: .zero, width: 100, height: 100)
+        )
+        documentViewButton.titleLabel?.adjustsFontSizeToFitWidth = true
+
+        documentViewButton.center = .init(x: 100, y: 100)
+        documentViewButton.addTarget(self, action: #selector(didTapDocumentButton), for: .touchUpInside)
+        view.addSubview(documentViewButton)
+    }
+
+    private func addPdfView() {
+        view.addSubview(pdfView)
+        guard let url = Bundle.main.url(forResource: "Lab01Qns", withExtension: "pdf") else {
```

Comment:
If this is a sample can you call it from SampleData. E.g. `let url = SampleData().getSamplePdfUrl()`. Also since you already added 5 sample urls, I think you can start linking things up properly.

So if they click on a document in the list, it should load the pdf of that document.
If they click on the add button, since it's a new document space, the pdf upload space should be empty with a button in the middle for them to upload a pdf.